configurations/arm/ubuntu2204.yaml

```diff
@@ -0,0 +1,8 @@
+ami_db_scylla_user: 'ubuntu'
```

Comment:
if we will not support ARM, please remove this file

Reply:
@fgelcer , ARM is removed:
```
	deleted:    configurations/arm/ubuntu2204.yaml
	deleted:    jenkins-pipelines/artifacts-ubuntu2204-arm.jenkinsfile
```